frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -59,7 +59,9 @@ Cypress.Commands.add('testA11y', (target: string) => {
 export const checkErrors = () =>
   cy.window().then((win) => {
     if (win.windowError) {
-      throw new Error(`window/js runtime error detected: ${win.windowError}`);
```

Comment:
Will the test still fail if we remove the `throw`? Should it just be `throw win.windowError`?

Reply:
not sure, we can see what it looks like in CI